db/migrate/20160825213523_change_load_balancer_listener_port_to_port_range.rb

```diff
@@ -0,0 +1,55 @@
+class ChangeLoadBalancerListenerPortToPortRange < ActiveRecord::Migration[5.0]
+  def change
+    reversible do |dir|
```

Comment:
I didn't realize the standard was to use def up/down.  I suppose reversible is slightly less code, but if it's confusing to read I'm happy to fix.